devtools/x/src/generate_summaries.rs

```diff
@@ -18,6 +18,9 @@ pub struct Args {
     #[structopt(name = "OUT_DIR")]
     /// Directory to output summaries to (default: target/summaries)
     out_dir: Option<PathBuf>,
+    #[structopt(name = "OUTPUT_FORMAT")]
+    /// Output in text, toml or json
+    output_format: Option<String>,
```

Comment:
This should be an enum using `arg_enum!`, here's an example: https://github.com/diem/diem/blob/39e4d7a064ea4b9d071c56d9df5099b655c4b8f8/devtools/x/src/cargo/build_args.rs#L7

In general, input validation should occur as close to the edges of the program as possible.

Reply:
done!